docs/index.html

```diff
@@ -17,20 +17,21 @@
     />
 
     <style>
-      :root {
-        /* Change the theme color hue */
-        --theme-hue: 300;
-      }
+        :root {

```

Comment:
Why did the indentation change here?
Seems like the rest of the file is 2 spaces instead of 4.

Reply:
Looks like this is due to incorrect css within the style block; there is a superfluous closing bracket ("}") apparently resulting in prettier increasing indentation here instead of giving an error.